chemprop/nn/metrics.py

```diff
@@ -74,8 +74,20 @@ class MSEMetric(MSELoss, Metric):
 
 @MetricRegistry.register("rmse")
 class RMSEMetric(MSEMetric):
+    def __call__(
+        self,
+        preds: Tensor,
+        targets: Tensor,
+        mask: Tensor,
+        w_s: Tensor,
+        w_t: Tensor,
+        lt_mask: Tensor,
+        gt_mask: Tensor,
+    ):
+        return self.forward(preds, targets, mask, lt_mask, gt_mask)[mask].mean().sqrt()
+
     def forward(self, *args, **kwargs) -> Tensor:
-        return super().forward(*args, **kwargs).sqrt()
+        return super().forward(*args, **kwargs)
```

Comment:
you can delete this function entirely if it only calls the `super()`

Reply:
Got it. I have removed the forward function() and implemented the call to `super()` in the '__call__()' method itself.